### PR TITLE
New Style: "Classic"

### PR DESCRIPTION
**Pull request by Parasite42**
The Classic style clearly apes the radar style that has been used in STUI's radar design.
The design is based upon the 'BI' style with the following changes made:
* Outlines on the direction characters have been removed (as well as a few subtle changes to the shape of the type)
* Radar Backgrounds have been changed from light grey to black
* Radar Outlines and bounds have been colourised as black
* _Radar backgrounds have been extended slightly to properly cover their respective zones_

**"Why did you make this and why couldn't you just use STUI you fucking autist?"**
DUI integrates a variety of very useful readability and personal preference features into its Radar suite. its a step up from STUI but calling it a logical next step is overstating it. Personally speaking though, I had a dislike for the BI Radar style as the brightening effect threw me off a little during my unit's OP on Tuesday (don't get me wrong though, it works well with the "just the tip" icon style but purely white icons against a white background doesn't work well, especially when JIPs or lost squad members start filling up your squad). I have made these changes and experimented with them in a private build of the mod that effectively replaces the 'BI' style (I'm too butfuck retarded to learn how to properly build something). I don't really have any other good reasons for suggesting this style (or even using this mod over STUI (yeah its a bit rude to say that).)
example 3
This fork was built upon an older version of DUI but hopefully it can be intergrated properly if it is accepted. Thanks for reading

### CODE REVIEW

File: mod/compass_style.hpp [hunk 1]
```diff
@@ -34,4 +34,11 @@ class diwako_dui_compass_style
         full = "diwako_dui\UI\compass_styles\clock\full.paa";
         limited = "diwako_dui\UI\compass_styles\clock\limited.paa";
     };
+    
+        class classic
```
Comment (Drofseh) [inline mod/compass_style.hpp]: wrong indentation

**Comment by diwako** [comment]
Thanks for the PR, I'll take a look once I get the time (after work). As @Drofseh already mentioned, there is a wrong indentation in the config.

While that is being corrected you can also add yourself to the authors file, so you are properly credited as well. :)

**Comment by Parasite42** [comment]
I should have hopefully changed both by now if I did everything correctly. Feel free to correct me if i screwed up somewhere.

**Comment by diwako** [comment]
Thanks. Looking goof so far. Will make an in depth review once I am home.

**Pull request merged**